tests/test_api.py

```diff
@@ -0,0 +1,278 @@
+# coding=utf-8
+"""Unit tests for :mod:`integrade.api`."""
+import json
+from json import JSONDecodeError
+from unittest import mock
+from unittest.mock import MagicMock
```

Comment:
I can see below usage like `mock.Mock` and `MagicMock` I think it would be better to be consistent and maybe import mock and use `mock.Mock` and `mock.MagicMock`

Reply:
Or just import all the members of `unittest.mock` we need here (`Mock`, `MagicMock`, and `patch`) since `mock` isn't ever being used directly and since importing the members themselves will give us shorter lines of code.